HookTest/cmd.py

```diff
@@ -65,6 +85,11 @@ def parse_args(args):
         default=False
     )
 
+    parser.add_argument(
+        "--guidelines", help="The version and type of guidelines to use",
+        choices=("2.tei", "2.epidoc")
```

Comment:
Add a default to 2.tei hgere :)

Reply:
If we want to determine the guidelines based on the scheme (epidoc or tei) then I don't think we should have a default. Instead, it should return None so that we can set the guidelines depending on scheme.

Reply:
You are totally right.